include/cxxopts.hpp

```diff
@@ -1844,6 +1847,13 @@ Options::parse(int& argc, char**& argv)
 inline
 void
 ParseResult::parse(int& argc, char**& argv)
+{
+  parse(argc, const_cast<const char**&>(argv));
```

Comment:
I don't think you need this. You only need const cast to take away constness. Adding constness should never be a problem.

Reply:
I'm not sure if I understand, could you elaborate? - Passing a 'char**' to a method that takes 'const char**' doesn't compile, hence the const_cast. E.g. reinterpret_cast will not work because "reinterpret_cast from type 'char**' to type 'const char**' casts away qualifiers".

Reply:
It compiled fine when I took away the `const_cast`. You can always pass non-const to const, it's the other way around that you can't do.

Reply:
In fact you can take the `char**&` version away and leave the `const char**&` version.

Reply:
Fixed in 216f2ca